src/index.js

```diff
@@ -0,0 +1,99 @@
+const state = {
+    tempCount: 0
+  };
+
+// temperature change function
+const tempchanger = (temp) => {
+    if (temp >= 80) {
+        return ["red", "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"];
+    } else if (temp > 70 && temp < 80) {
+        return ["orange", "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"];
+    } else if (temp > 60 && temp < 70) {
+        return ["yellow", "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"];
+    } else if (temp > 50 && temp < 60) {
+        return ["green", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"];
+    } else if (temp < 50) {
+        return ["teal", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"];
+    }
+
+}
+
+// Sky change function
+const skyChanger = (sky) => {
+  if (sky === "sunny") {
+    return "☀️";
+  } else if (sky === "cloudy") {
+    return "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+  } else if (sky === "rainy") {
+    return "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+  } else if (sky === "snowy") {
+    return "🌧🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+  } 
+}
+
+
+
+// Up Temp
+const uptemp = (event) => {
+    state.tempCount += 1;
+    const temp = document.querySelector("#tempCount")
+    const div = document.getElementById("tempCount")
+    tempData = tempchanger(state.tempCount)
+    div.style.color = tempData[0]
+    temp.textContent = `${state.tempCount}`
+    document.getElementById("wetherEmoji").innerHTML = tempData[1];
+
+};
+
+
+// Down Temp
+const downtemp = (event) => {
```

Comment:
This function is the same as uptemp aside from the first line and is a great candidate for a helper function.